ntbs-service/wwwroot/css/_colors.scss

```diff
@@ -1,47 +1,58 @@
-$phe-red: #822433;
-$phe-red-dull: #610925;
-$phe-navy: #002776;
-$phe-yellow: #eaab00;
+$dark-blue: #003B5C;
+$ukhsa-teal-dull: #005665;
+$ukhsa-teal: #007C91;
+$ukhsa-tangerine: #FF7F31;
+$ukhsa-sunny: #FFB81C;
 
 $grey: #999999;
 $lighter-grey: #CCCCCC;
-$shaded-background-color: #F0F4F5;
+$light-greyish-cyan: #F0F4F5;
 
-$primary-button: $phe-red;
-$primary-button-focus: $phe-red-dull;
+
+$application-brand: $ukhsa-teal;
+$application-brand-dull: $ukhsa-teal-dull;
+$ref-lab-header: $dark-blue;
+$secondary-button-color: $grey;
+$border-color: $grey;
+$border-colour-lighter: $lighter-grey;
+$shaded-background-color: $light-greyish-cyan;
+$focus: $ukhsa-sunny;
+$legacy-border: $ukhsa-tangerine;
+
+$primary-button: $application-brand;
+$primary-button-focus: $application-brand-dull;
 $primary-button-drop-shadow: shade($primary-button, 50%);
-$secondary-button: #425563;
-$secondary-button-focus: #384854;
+$secondary-button: $secondary-button-color;
+$secondary-button-focus: #7a7a7a;

```

Comment:
Do you think it would be worth naming all colours - so that they're all together in one place and it's clear how many there are? Or do you think that's overkill?

Reply:
The colours that are left unnamed are one that are only used in one instance, but I agree the file would be neater if they were all named at the top. I'll make the change and then we can decide!